ibmcloud/image/build.sh

```diff
@@ -34,6 +34,17 @@ if [[ -z "${base_img_path-}" || -z "${dst_img_path-}" || -z "${files_dir-}" ]];
     exit 1
 fi
 
+SE_BOOT=${SE_BOOT:-0}
+
+if [ "${SE_BOOT}" = "1" ]; then
+    if [[ -z "${HOST_KEYS_DIR-}" ]]; then
+        echo "HOST_KEYS_DIR is missed" 1>&2
+        echo "CLOUD_PROVIDER=ibmcloud SE_BOOT=1 HOST_KEYS_DIR=<host keys directory> make build"
+        exit 1
+    fi
+    rm -rf rootkeys
```

Comment:
If the build fails for any reason prior to the `shred`, `umount` and `rm` on line 362.
So I think adding something similar here and in the cleanup function (maybe line 72?):
```
umount ./rootkeys/ || true
rm -rf ./rootkeys/
```

Reply:
updated.

Reply:
Please could we add it at line 44 as well?

Reply:
added